client/src/components/ratingsReviews/ratingProgress.css

```diff
@@ -0,0 +1,19 @@
+meter {
+  display: block;
+  --background: #CCCCCC;
+  --optimum: rgb(36, 155, 36);
+}
+
+meter::-webkit-meter-bar {
+  background: var(--background);
+}
+
+.progress:hover {
+  cursor: pointer;
+  background-color: yellow;
+}
+
+label:hover {
```

Comment:
Along the same lines as my previous comment, anyone styling a `label` element without specifying a class name will either be locked into your styling or lock you into theirs.

Reply:
Updated CSS class to .progress-bar.label. Will check the rest of my css files for similar refactors.